internal/provider/resources/resource_team.go

```diff
@@ -160,17 +160,21 @@ func (r *TeamResource) Create(
 		return
 	}
 
-	memberIds, diags := utils.TypesSetToStringSlice(ctx, data.MemberIds)
-	if diags.HasError() {
-		resp.Diagnostics.Append(diags...)
-		return
+	var memberIdsPtr *[]string
+	if !data.MemberIds.IsNull() {
+		memberIds, diags := utils.TypesSetToStringSlice(ctx, data.MemberIds)
+		if diags.HasError() {
+			resp.Diagnostics.Append(diags...)
+			return
+		}
+		memberIdsPtr = &memberIds
```

Comment:
This allows us to explicitly send empty string list when a user passes in `memberIds = []` and null when `memberIds` is not provided. Previously we were always sending `[]string{}` for both cases, causing us to have conversion issues in the `ReadFromResponse` step.

Reply:
what would be the difference between `null` and `[]` on the api side though?